packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -276,3 +420,29 @@ class _AndroidPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+class _UiKitPlatformView extends LeafRenderObjectWidget {
+  const _UiKitPlatformView({
+    Key key,
+    @required this.viewId,
+    @required this.hitTestBehavior,
+  }) : assert(viewId != null),
+       assert(hitTestBehavior != null),
+       super(key: key);
+
+  final int viewId;
+  final PlatformViewHitTestBehavior hitTestBehavior;
+
+  @override
+  RenderObject createRenderObject(BuildContext context) =>
+      RenderUiKitView(
+        viewId: viewId,
+        hitTestBehavior: hitTestBehavior,
+      );
```

Comment:
nit: avoid line breaks in `=>` expressions

Reply:
done